Sources/ArcGISToolkit/Components/AR/FlyoverSceneView.swift

```diff
@@ -21,12 +21,14 @@ public struct FlyoverSceneView: View {
     @StateObject private var session = ObservableARSession()
     /// The initial camera.
     private let initialCamera: Camera
-    /// A Boolean value indicating whether to orient the scene view's initial heading to compass heading.
-    private let shouldOrientToCompass: Bool
     /// The translation factor.
     private let translationFactor: Double
+    /// A Boolean value indicating whether to orient the scene view's initial heading to compass heading.
+    private let shouldOrientToCompass: Bool
     /// The closure that builds the scene view.
     private let sceneViewBuilder: (SceneViewProxy) -> SceneView
+    /// The configuration used for the AR session.
+    private let configuration: ARConfiguration
```

Comment:
I don't think we need this. We just need to create it in onAppear and pass it to start(...)